vendor.conf

```diff
@@ -130,14 +130,14 @@ github.com/googleapis/gax-go                        bd5b16380fd03dc758d11cef74ba
 google.golang.org/genproto                          3f1135a288c9a07e340ae8ba4cc6c7065a3160e8
 
 # containerd
-github.com/containerd/containerd                    d4e78200d6da62480c85bf6f26b7221ea938f396
-github.com/containerd/fifo                          f15a3290365b9d2627d189e619ab4008e0069caf
+github.com/containerd/containerd                    0edc412565dcc6e3d6125ff9e4b009ad4b89c638 # master (v1.5.0-dev)
```

Comment:
Could you update this to be newer than https://github.com/containerd/containerd/commit/7126310a09500e1278f62fd1c12b31177a6434ff ?
I believe the older commit is safe for using a library, but the older daemon binary was known not to work with Docker, so I'm not a fan of vendoring the older commit.

Can be a follow-up PR though.